functions.php

```diff
@@ -76,3 +76,12 @@
 	add_action( 'wp_ajax_text_diff', '\PressbooksBook\Actions\text_diff' );
 	add_action( 'wp_ajax_nopriv_text_diff', '\PressbooksBook\Actions\text_diff' );
 }
+
```

Comment:
Probably want to move this into a helper class if we keep it -- it's here just so you can add the `?lti_view` param to content to see the stripped down view in your browser.